README.md

```diff
@@ -137,17 +133,25 @@ The `config` input is a YAML string which describes your terraform workspaces.
 The workspaces are grouped by account in order to de-duplicate some shared settings for the common scenario of dev/prod accounts.
 The root of the YAML document should be an array of `Account` objects.
 
+###### `Account.account_id` (`string`)
+
+**Optional**.
```

Comment:
Mayhaps?

```suggestion
**Recommended**.
```

Reply:
Yeah that's better. I wondered if I should just make it "required". "Recommended" is more accurate and also what we want.